### ISSUE TITLE
training partition detail

### ISSUE THREAD

**Issue by sc-56** [xu-ji/IIC#73]
Sorry for bothering again, 

I'm trying to create my own dataset for the training, and may I roughly know the detail of the training partitions (how many partitions, and is that used for cross-value validation?) 

Thank you so much, sorry again for coutinuous questions.

**Comment by xu-ji** [comment]
It's your decision. For fully unsupervised learning it's accepted to have training data = test data for example. We didn't do cross-validation.

The mention of `partitions` in the code is mainly for me to distinguish between the fully unsupervised and semi-supervised case, as these have different requirements. You may not need any `config.*partitions` variables if you wrote your own dataset code.

For fully unsupervised, in general you can just make all partitions all your data. The only dataset where head A and B see different data is STL10, because some of the data is known to be less relevant (explained in paper). But the `*mapping*` dataloaders need to have only the labelled data, because they're used in accuracy evaluation.

**Comment by sc-56** [comment]
Thank you so much for your detailed explanation.
